test/CMakeLists.txt

```diff
@@ -5,4 +5,5 @@ endif(POLICY CMP0037)
 add_custom_target(test
     COMMAND ${PROJECT_SOURCE_DIR}/test/test-run.py -j -1
         --builddir=${PROJECT_BINARY_DIR}
-        --vardir=${PROJECT_BINARY_DIR}/test/var)
+        --vardir=${PROJECT_BINARY_DIR}/test/var
+        --force)
```

Comment:
In tarantool we have two make targets: `make test` and `make test-force`. I don't know whether we actully should follow this way, just highlighted that if we'll adjust `make test` in vshard, we'll lack consistency with tarantool here.

Reply:
Thanks for your remark! Fixed.